plugins/dspam/dspam_

```diff
@@ -211,7 +215,15 @@ file_is_locked() {
 	local file=$1
 	local lock=$1.lock
 
-	[ -f $lock ] && debug file $file is locked && return 0 # EX_OK
+	if [ -f $lock ];then
+		local pid=$(cat $1.lock)
+		if [ $(ps h -p $pid -o comm=|grep -c dspam_) -eq 0 ]; then
+			debug lock for file $file is no longer valid
+			file_remove_lock $file
+		else
+			debug file $file is locked && return 0 # EX_OK
```

Comment:
Maybe split both commands?
This will make the flow-breaking `return` a bit more obvious.
Additionally it removes the conditional relationship between these (unrelated) commands.